code/modules/mob/living/simple_animal/hostile/wendigo.dm

```diff
@@ -0,0 +1,221 @@
+/*	TODO:
+	Let them evolve and heal from eating. Should a carbon they eat go over a certain damage threshold and
+	the wendigo is at full health. Strip and delete the corpse.
+	Human wendigo->1 eat->Wendigo->3 eat->Alpha Wendigo (Can only be ONE at any time)
+	They don't like fire. Will stay away from fire (As far as their light radius -1) and won't target anyone
+	in vicinity of said fire
+	Can mimic voices, steal poly code, and change it (Maybe just keep a log of people consumed,
+	and use their name plus shout for help)
+	Butcherable for one piece of meat only. When consumed (Don't use reagents) will begin to turn the person
+*/
+#define HUMEVOLV 1
```

Comment:
this is gonna sound petty but it's not entirely clear what these stand for

Reply:
far as I could tell, they're defines for the specific wendigo evolutions. HUMEVOLV is for the human looking one, and the other's for the weird and silly looking one.

Big one doesn't evolve, so no define for it.